src/Xamarin.Forms.PancakeView.Multi/Platforms/UWP/PancakeViewRenderer.cs

```diff
@@ -117,30 +132,55 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
             {
                 UpdateBorder(pancake);
             }
+
+            else if (e.PropertyName == PancakeView.WidthProperty.PropertyName ||
+                    e.PropertyName == PancakeView.HeightProperty.PropertyName)
+            {
+                UpdateShadow(pancake);
+            }
         }
 
         private void UpdateShadow(PancakeView pancake)
         {
-            if (Control != null)
+            //For now gets the shadow only when the CornerRadius has the same value for all sides. 
+            if (Control != null && pancake.HasShadow && pancake.Width > 0 && pancake.Height > 0 &&
+                pancake.CornerRadius.TopLeft == pancake.CornerRadius.BottomLeft &&
+                pancake.CornerRadius.BottomRight == pancake.CornerRadius.TopRight)
             {
-
+                rectangle.Fill = new SolidColorBrush(Windows.UI.Colors.Black);

```

Comment:
Would this work with transparent as a color so we're sure no black shines through? The sample app looks like this for me:

![image](https://user-images.githubusercontent.com/2419439/74849467-062bda80-5339-11ea-997d-a2745e04a792.png)

Reply:
It is not working for combination with individual rounded corners yet. I'll check about to use transparent color.

Reply:
I think it's better to not apply the shadow at all unless all 4 radii are the same. I will update the documentation accordingly to clearly state that limitation. Should be a minor adjustment to the current `if` check that's already in place.

Reply:
Other than that it's fine! Thanks again for making this :)